AllReadyApp/Web-App/AllReady/Areas/Admin/Features/Invite/CreateInviteCommandHandler.cs

```diff
@@ -0,0 +1,90 @@
+﻿using AllReady.Areas.Admin.ViewModels.Invite;
+using AllReady.Models;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace AllReady.Areas.Admin.Features.Invite
+{
+    public class CreateInviteCommandHandler : AsyncRequestHandler<CreateInviteCommand>
+    {
+        private AllReadyContext _context;
+
+        public CreateInviteCommandHandler(AllReadyContext context)
+        {
+            _context = context;
+        }
+
+        public Func<DateTime> DateTimeUtcNow = () => DateTime.UtcNow;
+
+        protected override async Task HandleCore(CreateInviteCommand message)
+        {
+            if (message?.Invite == null) throw new ArgumentException("InviteViewModel must be set");
+            if (message?.UserId == null) throw new ArgumentException("User Id must be set");
```

Comment:
Remove both of these checks. We don't do field validation in handlers, that type of validation should be delegated to controllers. Who is sending us the command? So we trust that controller has done it's job and validated the fields for us